Client/Ecosia/UI/Ecosia.xcassets/ecosiaLogoLaunch.imageset/Contents.json

```diff
@@ -11,7 +11,7 @@
           "value" : "dark"
         }
       ],
-      "filename" : "Ecosia logo.pdf",
+      "filename" : "Ecosia logo (1).pdf",
```

Comment:
Out of curiosity, do we have different logos? And what would be the difference between naming "Ecosia logo-1.pdf" and "Ecosia logo (1).pdf"?

Reply:
@jman82 That is a very good question.

There is no difference at all, it is not easily spotted here on GitHub, but on Xcode, when opening an Asset Catalog, for example `Ecosia.xcassets`, you can find an image named `ecosiaLogoLaunch`, which is the one being updated here.

**ecosiaLogoLaunch** is the name used by the application for this image, internally the file can be named differently, and in most cases will actually be named differently. For example in this case we have 2 images, 1 for dark and 1 for light mode. Both of them need to have different names to coexist on the same directory, but they get accessed with the same name on the application, the system chooses which one to pick up according to the current environment. There can be cases where instead of 2 images there are 9 or more living under the same name, or could only be 1 also.

Let me know if this clarifies more or less 👍 .

Reply:
Awesome! Thank you very much for explaining me @vauxhall :tropical_drink: :+1: